packages/storybook/stories/Dialpad/snippets/CustomDialpad.snippet.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Dialpad, DialpadButtonsType } from '@azure/communication-react';
+import React from 'react';
+
+const dialPadButtons: DialpadButtonsType[][] = [
+  [
+    { primaryContent: '0' },
+    { primaryContent: '0', secondaryContent: '123' },
+    { primaryContent: '0', secondaryContent: '456' }
+  ],
+  [
+    { primaryContent: '+', secondaryContent: '789' },
+    { primaryContent: '+', secondaryContent: '012' },
+    { primaryContent: '+', secondaryContent: '345' }
+  ],
+  [
+    { primaryContent: 'q', secondaryContent: '678' },
+    { primaryContent: 'q', secondaryContent: '098' },
+    { primaryContent: 'q', secondaryContent: '111' }
+  ],
+  [
+    { primaryContent: '*', secondaryContent: '678' },
+    { primaryContent: '0', secondaryContent: '+' },
+    { primaryContent: '#', secondaryContent: '678' }
+  ]
+];
+
+export const CustomDialpadExample: () => JSX.Element = () => {
```

Comment:
This is a great example! However found that because we sanitize the input to be numbers only, if the primary content is a `q` or any letter it won't work when clicked
![image](https://user-images.githubusercontent.com/2684369/167668919-02337810-8eba-4f7d-87e8-66aec9014930.png)

Reply:
Updated and added note in storybook mentioning not allow letters and some special characters as input